src/services/search.service.ts

```diff
@@ -6,20 +6,21 @@ import { CipherService } from '../abstractions/cipher.service';
 import { LogService } from '../abstractions/log.service';
 import { SearchService as SearchServiceAbstraction } from '../abstractions/search.service';
 
+import { I18nService } from '../abstractions';
```

Comment:
Please move this up to the other `abstractions` block above this block, in alphabetical order (so between lines 5 and 6 above). 

Also, missed this last time, you should import the actual service, we don't use general directory imports, so `from '../abstractions/i18n.service';` instead of just `from '../abstractions';`.